webviews/editorWebview/index.css

```diff
@@ -468,11 +468,41 @@ body button .icon {
 #status {
 	box-sizing: border-box;
 	line-height: 18px;
-	background: var(--vscode-badge-background);
-	color: var(--vscode-badge-foreground);
+	color: var(--vscode-foreground);
```

Comment:
This renders the text as gray on top of the color, which doesn't look right. `editorInlayHint-foreground` is probably suitable here since that's what I'm doing to get the brighter text on the branch name labels below.

Reply:
Scratch that, it doesn't always work unless used with its background color counterpart. `button-foreground` is worth a try.